pyaedt/generic/general_methods.py

```diff
@@ -1327,6 +1327,78 @@ def compute_fft(time_vals, value):  # pragma: no cover
     return freq, valueFFT
 
 
+@pyaedt_function_handler()
+def conversion_function(data, function_str=None):  # pragma: no cover
+    """This method performs various types of conversions on the input data based on the specified function string.
```

Comment:
```suggestion
    """Convert input data based on a specified function string.
    
```
Method descriptions start with a simple verb. Details can follow this brief description following a blank line.